### PR TITLE
Delete metadata for transient keys

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
This requires re-generating a key on import, which is weird, I'll see if there are ways to avoid that

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -42,7 +42,7 @@ const BACKEND_DIR: &Path = path!("se050-bak");
 pub const GLOBAL_ATTEST_ID: ObjectId = ObjectId(hex!("F0000012"));
 
 /// The version to know wether it should be re-configured
-pub const SE050_CONFIGURE_VERSION: u32 = 2;
+pub const SE050_CONFIGURE_VERSION: u32 = 3;
```
Comment (robin-nitrokey) [inline src/lib.rs:45]: Why do we need a new version?  AFAIS `REQUIRED_CURVES` did not change.

Comment (sosthene-nitrokey) [reply src/lib.rs:45]: This PR removes the `large-ecc` feature  so they're now always enabled. Worst case I don't think it's a problem to bump the version.

Comment (robin-nitrokey) [reply src/lib.rs:45]: But the feature did not affect curve initialization.  See also:  https://github.com/Nitrokey/trussed-se050-backend/pull/33#discussion_r1698125710

Comment (sosthene-nitrokey) [reply src/lib.rs]: Right, removed.

**Comment by sosthene-nitrokey** [comment]
Ok, this is fixed, there  were some challenges with private-key only imported keys (brainpool and nist curves larger than 256 bits), since the re-import would not accept not having the exact same metadata, but that should be fixed now!

The full opcard test suite passes on hardware including brainpool and large nist curves now!

**Review by robin-nitrokey — COMMENTED** [review]
The documentation for `reimport_volatile_key` still needs to be updated (`reselect` has been removed, maybe mention re-creation), otherwise LGTM.

**Review by robin-nitrokey — APPROVED** [review]

**Pull request merged**